doc/HOW_TO _RUN_BUILDL.md

```diff
@@ -0,0 +1,20 @@
+# Welcome to Explanation of the bash script buildl.sh 
+## (1) First lets take a look in the usage of the script:
+		usage: ./buildl.sh [test_0] [test_1] ... [test_n]
```

Comment:
Your editor is using TABs?
Please make sure its changed to Spaces.
See what it looks like here:(bad alignment).
![image](https://user-images.githubusercontent.com/81047407/147110946-eac03187-b9cf-4d65-90ab-f289c0895bd1.png)

Reply:
Apparently this is a problem showing up here on github, it looks just fine in my environment.

Reply:
That is exactly why you should use  spaces and not tabs!! So it looks the same in all environment!!

Please remove the tabs and use spaces

Reply:
That is exactly why you should use  spaces and not tabs!! So it looks the same in all environment!!

Please remove the tabs and use spaces

Reply:
Okay, I got it. The issue will be addressed.